source/_components/climate.isy994.markdown

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
You can add the information on this page to the component page, delete this one.